File: Python/1154.py
class Solution(object):
    def dayOfYear(self, date):
        year,month,day = map(int,date.split("-"))
        lst = [31,28,31,30,31,30,31,31,30,31,30]
        count = day
        for i in range(month-1):
            count += lst[i]
        if year%4 == 0 and year != 1900 and month>2:
            count += 1
        return count

File: Java/1154.java
public class Solution {
    public int dayOfYear(String date) {
        int[] arr = {31,28,31,30,31,30,31,31,30,31,30,31};
        int year = Integer.valueOf(date.substring(0,4));
        int month = Integer.valueOf(date.substring(5,7));
        int day = Integer.valueOf(date.substring(8));
        for(int i = 0; i < month-1; i++) day += arr[i];
        if(year%4 == 0 && year != 1900 && month > 2) day++;
        return day;
    }
} 